i18n(it): Updates `reference/configuration.mdx` (#2195)
withastro/starlight: docs/src/content/docs/it/reference/configuration.mdx

Author: certanly, will fix in next commit

### `tableOfContents`

**tipo:** `false | { minHeadingLevel?: number; maxHeadingLevel?: number; }`  

> **default:** `{ minHeadingLevel: 2; maxHeadingLevel: 3; }`

Author: Must have slipped by... sorry :)

**tipo:** `string`

Imposta la lingua predefinita per questo sito.
Il valore dovrebbe corrispondere a una delle chiavi del tuo oggetto [`locales`](#locales).

> (Se la tua lingua predefinita è il tuo [locale radice](#locale-radice), puoi saltare questo.)

Reviewer: ```suggestion
(Se la tua lingua predefinita è il tuo [locale radice](#lingua-principale), puoi saltare questo.)
```

**tipo:** `StarlightExpressiveCodeOptions | boolean`
**predefinito:** `true`

Starlight utilizza [Expressive Code](https://github.com/expressive-code/expressive-code/tree/main/packages/astro-expressive-code) per renderizzare i blocchi di codice e aggiungere supporto per evidenziare parti di esempi di codice, aggiungere nomi di file ai blocchi di codice e altro ancora.

> Consulta la [guida "Blocchi di codice"](/guides/authoring-content/#code-blocks) per imparare come utilizzare la sintassi di Expressive Code nel tuo contenuto Markdown e MDX.

Reviewer: ```suggestion
Consulta la [guida "Blocchi di codice"](/it/guides/authoring-content/#blocco-di-codice) per imparare come utilizzare la sintassi di Expressive Code nel tuo contenuto Markdown e MDX.
```

The previous link was correct.

Consulta la [documentazione di Expressive Code sui `themes`](https://github.com/expressive-code/expressive-code/blob/main/packages/astro-expressive-code/README.md#themes) per i dettagli sui formati di tema supportati.

Starlight utilizza di default le varianti scura e chiara del tema [Night Owl](https://github.com/sdras/night-owl-vscode-theme) di Sarah Drasner.

Se fornisci almeno un tema scuro e uno chiaro, Starlight manterrà automaticamente sincronizzato il tema attivo del blocco di codice con il tema corrente del sito.
Configura questo comportamento con l'opzione [`useStarlightDarkModeSwitch`](#usestarlightdarkmodeswitch).

#### `useStarlightDarkModeSwitch`

**tipo:** `boolean`
**predefinito:** `true`

Quando `true`, i blocchi di codice passano automaticamente tra i temi chiari e scuri quando cambia il tema del sito.
Quando `false`, devi aggiungere manualmente CSS per gestire il passaggio tra più temi.

:::note
Quando imposti `themes`, devi fornire almeno un tema scuro e uno chiaro affinché il cambio di modalità scura di Starlight funzioni.
:::

#### `useStarlightUiThemeColors`

**tipo:** `boolean`
**predefinito:** `true` se `themes` non è impostato, altrimenti `false`

> Quando `true`, le variabili CSS di Starlight vengono utilizzate per i colori degli elementi dell'interfaccia utente dei blocchi di codice (sfondi, pulsanti, ombre, ecc.), corrispondenti al [tema di colore del sito](/guides/css-and-tailwind/#theming).

Reviewer: ```suggestion
Quando `true`, le variabili CSS di Starlight vengono utilizzate per i colori degli elementi dell'interfaccia utente dei blocchi di codice (sfondi, pulsanti, ombre, ecc.), corrispondenti al [tema di colore del sito](/it/guides/css-and-tailwind/#temi).
```

**tipo:** `boolean`  
**predefinito:** `false`

Controlla se il footer mostra quando la pagina è stata aggiornata l'ultima volta.

> Per impostazione predefinita, questa funzionalità si basa sulla cronologia Git del tuo repository e potrebbe non essere accurata su alcune piattaforme di distribuzione che eseguono [cloni superficiali](https://git-scm.com/docs/git-clone#Documentation/git-clone.txt---depthltdepthgt). Una pagina può sovrascrivere questa impostazione o la data basata su Git utilizzando il [campo frontmatter `lastUpdated`](/reference/frontmatter/#lastupdated).

Reviewer: ```suggestion
Per impostazione predefinita, questa funzionalità si basa sulla cronologia Git del tuo repository e potrebbe non essere accurata su alcune piattaforme di distribuzione che eseguono [cloni superficiali](https://git-scm.com/docs/git-clone#Documentation/git-clone.txt---depthltdepthgt). Una pagina può sovrascrivere questa impostazione o la data basata su Git utilizzando il [campo frontmatter `lastUpdated`](/it/reference/frontmatter/#lastupdated).
```

### `pagination`

**tipo:** `boolean`  
**predefinito:** `true`

Definisce se il footer deve includere i link alla pagina precedente e successiva.

> Una pagina può sovrascrivere questa impostazione o il testo del link e/o l'URL utilizzando i campi frontmatter [`prev`](/reference/frontmatter/#prev) e [`next`](/reference/frontmatter/#next).

Reviewer: ```suggestion
Una pagina può sovrascrivere questa impostazione o il testo del link e/o l'URL utilizzando i campi frontmatter [`prev`](/it/reference/frontmatter/#prev) e [`next`](/it/reference/frontmatter/#next).
```

### `favicon`

**tipo:** `string`  
**predefinito:** `'/favicon.svg'`

Imposta il percorso del favicon predefinito per il tuo sito web che dovrebbe essere posizionato nella directory `public/` ed essere un file di icona valido (`.ico`, `.gif`, `.jpg`, `.png`, o `.svg`).

```js
starlight({
  favicon: '/images/favicon.svg',
}),
```

> Se hai bisogno di impostare varianti aggiuntive o favicon di fallback, puoi aggiungere tag utilizzando l'[opzione `head`](#head):

Reviewer: Looks like the change from https://github.com/withastro/starlight/pull/1627 is missing in the code example below.

Author: will fix in the next commit
